lang/python/python-mysql/Makefile

```diff
@@ -1,48 +1,64 @@
 #
-# Copyright (C) 2007-2014 OpenWrt.org
+# Copyright (C) 2007-2018 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=MySQL-python
-PKG_VERSION:=1.2.5
+PKG_NAME:=python-mysql
+PKG_VERSION:=1.3.12
 PKG_RELEASE:=1
 PKG_LICENSE:=GPL-2.0
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).zip
-PKG_SOURCE_URL:=https://pypi.python.org/packages/source/M/MySQL-python/
-PKG_HASH:=811040b647e5d5686f84db415efd697e6250008b112b6909ba77ac059e140c74
+PKG_SOURCE:=mysqlclient-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/m/$(PKG_NAME)
+PKG_HASH:=2d9ec33de39f4d9c64ad7322ede0521d85829ce36a76f9dd3d6ab76a9c8648e5
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-mysql-$(PKG_VERSION)
 
 include $(INCLUDE_DIR)/package.mk
 include ../python-package.mk
+include ../python3-package.mk
 
-define Package/python-mysql
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+define Package/python-mysql/Default
   SUBMENU:=Python
   SECTION:=lang
   CATEGORY:=Languages
-  TITLE:=MySQL database adapter for Python
-  URL:=https://pypi.python.org/pypi/MySQL-python
+  URL:=https://pypi.python.org/project/mysqlclient
   MAINTAINER:=Gergely Kiss <mail.gery@gmail.com>
+endef
+
+define Package/python-mysql
+  $(call Package/python-mysql/Default)
+  TITLE:=MySQL database adapter for Python
   DEPENDS:=+python +libmysqlclient
```

Comment:
i recommend adding `+PACKAGE_python-mysql:python +PACKAGE_python-mysql:libmysqlclient`
similarly for `python3-mysql` add:  `+PACKAGE_python3-mysql:python +PACKAGE_python3-mysql:libmysqlclient`

otherwise if you build only `python-mysql` then `python3-mysql` will also get built and vice-versa; so it should save build time, because you won't build Python & Python3
this is a known behavior of the build-system;